### DISCUSSION TITLE
Unclear if volume mount cached mode applicable to Windows hosts

### DISCUSSION THREAD

**Discussion by ineffyble** [batect/batect#997] [Q&A] [answered]
Some docs pages such as the Containers config file reference suggest you can use "cached" with Windows for better performance.

However, the Improve Performance guide says MacOS hosts are the only ones that benefit

**Comment by charleskorn** [comment, marked as answer]
`cached` only applies to macOS hosts, and only if you're using the legacy (non-gRPC) file sharing ([source](https://github.com/docker/for-mac/issues/5402)). 

I'll update the docs to reflect this, thanks for letting me know.
